sbom/examples/rpm/release/add_release_data.py

```diff
@@ -80,6 +95,15 @@ def get_rpm_purl(ext_refs):
             }
             new_refs.append(release_ref)
 
+    for cpe in cpe_map[sbom_name]:
+        if purl.qualifiers["arch"] == "src":
+            product_ref = {
+                "referenceCategory": "SECURITY",
+                "referenceLocator": cpe,
+                "referenceType": "cpe22Type",
+            }
+            new_refs.append(product_ref)
+
```

Comment:
Product metadata is not included here on purpose. We add it in a separate example in https://github.com/RedHatProductSecurity/security-data-guidelines/blob/main/sbom/examples/product/rhel-9.2-eus.spdx.json.

Though now that I look at that example, I don't think we'll want to reduce the manifest to just its root element to add the product metadata to it at release time. Nor do we want the comprehensive manifest that is an aggregation of all the individual, component-level ones to be just SRPMs.

Can you modify this SBOM to add a product package like the above SBOM does it and add the CPEs there? And I'll remove that one and describe what a comprehensive product SBOM would look like in the docs.

@twaugh agree?

Reply:
I fixed the rpm/release examples to include a 'product' package filled in with variants from Errata Tool. I also added one extra relationships which is that the SRPM is a 'PACKAGE_OF' the 'product' package.

Reply:
@mprpic I'm a bit confused about why we're including product information in these at all, rather than the separate SBOMs in `../product`. Are we no longer wanting separate SBOMs for the product relationships?

What if an RPM is part of multiple products?

Reply:
You're right, we should continue using the separate product SBOM that links a product object with the main component of a component-level SBOM.

Related Slack thread: https://redhat-internal.slack.com/archives/C07MHR3MGLR/p1729197309219799

@jasinner Let's remove the product-specific metadata from the component-level manifests then. This data should only live in the separate manifest in `../product`.

Reply:
Ok, removed the product information from the release RPM manifests.